mediorum/server/serve_upload.go

```diff
@@ -119,7 +120,14 @@ func (ss *MediorumServer) postUpload(c echo.Context) error {
 	return c.JSON(200, uploads)
 }
 
-func (ss *MediorumServer) getV1CIDBlob(c echo.Context) error {
+func (ss *MediorumServer) getBlobByJobIDAndVariant(c echo.Context) error {
```

Comment:
this name could be more concise. the previous `getV1CIDBlob` name was confusing because it serves CID V1 and CID V2

Reply:
Yeah good call.  Have been thinking to put all the blob serving handlers in one file and change the names around after merging this and my other branch.